app/views/hyrax/base/_search_results.html.erb

```diff
@@ -8,7 +8,7 @@
 
 <%= render 'search_header' %>
 
-<h2 class="sr-only"><%= t('blacklight.search.search_results') %></h2>
+<div class="h2 sr-only"><%= t('blacklight.search.search_results') %></div>
```

Comment:
And we don't need to label these as search results, they should just look like a list of works